api/src/main/resources/workbench.yaml

```diff
@@ -84,6 +84,140 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  # Notebook clusters ####################################################################
+
+  /api/v1/clusters:
```

Comment:
Are we planning on showing the user a view of clusters they have running independent of any workspace? Or is it always "show me the status of the cluster for workspace X"?

Reply:
I am planning on always having it be: "show me the status of cluster for workspace X" but I think that's a UX question. It should definitely be possible to have a list of clusters the user has running be shown.